components/spi/kustomization.yaml

```diff
@@ -33,7 +33,7 @@ patches:
     path: oauth-service-deployment-patch.json
   - target:
       kind: ConfigMap
-      name: shared-config
+      name: shared-environment-config
```

Comment:
the configmap name is `spi-shared-config`. When I update it then I got SSL issues:
```
{"level":"error","ts":"2022-08-30T08:19:38Z","caller":"spi-oauth/main.go:126","msg":"failed to create token storage interface","error":"error while authenticating: unable to log in to auth method: unable to log in with Kubernetes auth: Put \"https://spi-vault-spi-system.apps.ci-ln-134278b-72292.origin-ci-int-gce.dev.rhcloud.com/v1/auth/kubernetes/login\": x509: certificate signed by unknown authority","stacktrace":"main.main\n\t/spi-oauth/main.go:126\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250"}
```